include/client/tickets.inc.php

```diff
@@ -47,10 +48,15 @@
 $$x=' class="'.strtolower($_REQUEST['order'] ?: 'desc').'" ';
 
 // Add visibility constraints
-$tickets->filter(Q::any(array(
+$visibility = Q::any(array(
     'user_id' => $thisclient->getId(),
     'thread__collaborators__user_id' => $thisclient->getId(),
-)));
+));
+
+if ($thisclient->canSeeOrgTickets())
+    $visibility->add(array('user__org_id' => $thisclient->getOrgId()));
```

Comment:
This block could be written like
```php
$visibility = Q::any(array(
    'thread__collaborators__user_id' => $thisclient->getId(),
));

if ($thisclient->canSeeOrgTickets())
    $visibility->add(array('user__org_id' => $thisclient->getOrgId()));
else
    $visibility->add(array('user_id' => $thisclient->getId());
```
which would help reduce the OR list in the WHERE clause and *should* help MySQL do less work to find the tickets